packages/eslint-plugin-template/src/utils/is-presentational-role.ts

```diff
@@ -1,10 +0,0 @@
-import type { TmplAstElement } from '@angular/compiler';
-import { PROPERTY } from './constants';
-import { getAttributeValue } from './get-attribute-value';
-
-const presentationRoles = new Set(['presentation', 'none', PROPERTY]);
-
-export function isPresentationRole(node: TmplAstElement): boolean {
```

Comment:
This file was named incorrectly and that's why I changed to `is-presentation-role`.